index.html

```diff
@@ -37,154 +36,250 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
+                    <div class="lang-text">Skills</div>
+                    <div class="">
+                    <ul>
+                    <div class="lang-text-list">
+                        <li class="li-lang" >CodeKit</li>
+                        <li class="li-lang">Github</li>
+                        <li class="li-lang">Codepen</li>
+                        <li class="li-lang">Gitlab</li>
+                        <li class="li-lang">Terminal</li>
+                    </div>
+                    </ul>
+                    </div>
+    
+                </article>
+                </div>
+            
+            </section>
+            </section>
 
-           </article>
-           <article class="project2">
-               <h3 class="language-icon"><img src="./images/icon-skills (1).png" alt=""></h3>
-               <h4 class="lang-text">Skills</h4>
-               <div class="">
-               <ul class="lang-text-list">
-                   <li class="li-lang" >CodeKit</li>
-                   <li class="li-lang">Github</li>
-                   <li class="li-lang">Codepen</li>
-                   <li class="li-lang">Gitlab</li>
-                   <li class="li-lang">Terminal</li>
-               </ul>
-               </div>
+            <section class="contact"> 
+            
+                <section class="form-section">
+                
+                    <div class="form-description">
+                    <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                    </div>
+                
+                    <div class="contact-form">
+                 <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                   
+                    <ul>
+                        
+                        <li >
+                            <label for="name">
+                                <input type="text" id="name"  name="user_name" placeholder="Full name" maxlength="30" required>
+                            </label>
+                        </li>
+                        <li>
+                            <label for="mail">
+                                <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                            </label>
+    
+                        </li>
+                        <li>
+                            <label for="msg">
+                                <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                            </label>
+                        </li>
+                        <li class="button back-image">
+                            <button type="submit" class="btn">Get in touch</button>
+                        </li>
+    
+                    </ul>
+    
+                </form>
+    
+                    </div>
+                </section>
 
+
+            </section>
```

Comment:
- [ ] I think you did not remark that this contact section is duplicated, please remove one and fix the styling before your project gets approved.
- [ ] The image on the side of the get in touch button should stick on the right side, please check the design and see where it is positioned.

![duplicate](https://user-images.githubusercontent.com/77843050/206151446-31d2b071-7746-4d49-aee0-a32b717e5578.gif)